src/test/java/jewellery/inventory/unit/service/security/JwtTokenServiceTest.java

```diff
@@ -230,4 +233,19 @@ private void setupDefaultExtractAllClaimsBehavior() {
 
     lenient().doReturn(claims).when(jwtUtils).extractAllClaims(anyString());
   }
+
+  private static Stream<String> invalidEmailProvider() {
+    return Stream.of(null, "", " ");
+  }
+
+  private String generateExpiredToken() {
+    Date expiredDate =
+        new Date(System.currentTimeMillis() - (1000 * 60 * 60 * 24)); // 1 day in the past
```

Comment:
Please, remove the comment. You can, instead, extract the complex calculation (1000 * 60 * 60 * 24) in a variable named dayInMillis.

Reply:
Fixed

Reply:
I see you ended up not extracting `dayInMillis = (1000 * 60 * 60 * 24)`. Any reason why not?